cldf/cldf-metadata.json

```diff
@@ -16,19 +16,19 @@
             "rdf:type": "prov:Entity",
             "dc:title": "Repository",
             "rdf:about": "https://github.com/tupian-language-resources/tuled",
-            "dc:created": "v0.10-120-g0c546c7"
+            "dc:created": "v0.10-122-g58fbf6f"
         },
         {
             "rdf:type": "prov:Entity",
             "dc:title": "Glottolog",
             "rdf:about": "https://github.com/glottolog/glottolog",
-            "dc:created": "v4.3"
+            "dc:created": "v4.2.1-13-g7970be034b"
```

Comment:
can you please re-run and add the keyword `--glottolog-version=v4.3`? The entry here suggests you run this without a specific version, and your glottolog repository seems to be outdated. Same holds for concepticon (please git-pull and use `--concepticon-version=v2.4.0`